.github/workflows/main.yml

```diff
@@ -2,7 +2,7 @@ name: Package
 
 on:
   push:
-    branches: [main, manifest-v3]
+    branches: [main, manifest-v3, testing]
```

Comment:
There is no testing branch in the repo

Reply:
yeah i was planning on removing it when everything was ready for merge